Notebooks/Bruxism_detection.ipynb

```diff
@@ -47,9 +47,9 @@
     "from tinnsleep.config import Config\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false"># remove the two first hours and last hour 
</pre><p>at the wrong place</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>